terraform/trial_rg/main.tf

```diff
@@ -115,15 +116,31 @@ resource "random_password" "roles_sql_password" {
 
 # Create a roles and permissions SQL
 # Deploy a sql server and db for fhir before we create the web app
-module "fhir_sql_server" {
+module "roles_sql_server" {
```

Comment:
Should we have a sql module that could be "turned" on for any service?

Reply:
currently two sql servers;
1. fhir
2. roles.


the second can have multiple dbs if we'd like so all services can access. wdyt?